server/aspects/core.bzl

```diff
@@ -69,9 +69,20 @@ def _get_forwarded_deps(target, ctx):
         return collect_targets_from_attrs(ctx.rule.attr, ["deps"])
     return []
 
+def _is_analysis_test(target):
+    """Returns if the target is an analysis test.
+
+    Rules created with analysis_test=True cannot create write actions, so the
+    aspect should skip them.
+    """
+    return AnalysisTestResultInfo in target
+
+
```

Comment:
This extra newline causes the formatter on CI to fail

Reply:
Thanks for the callout